package.json

```diff
@@ -7,7 +7,7 @@
     "packages/svg-icons"
   ],
   "scripts": {
-    "setup": "npm install && npx husky install",
+    "setup": "npm install && npx husky",
```

Comment:
Please add this to the `"prepare"` script, as this will automatically setup husky when you do a `npm ci`. See [ref](https://typicode.github.io/husky/get-started.html#husky-init-recommended).

You can also remove the `"setup"` script, as the repo setup now will consist of just running `npm ci` :)

@epetrow